grpclb/src/main/java/io/grpc/grpclb/GrpclbState.java

```diff
@@ -556,7 +549,8 @@ private void handleStreamClosed(Status error) {
       cleanUp();
       propagateError(error);
       balancerWorking = false;
-      maybeStartFallbackTimer();
+      maybeUseFallbackBackends();
```

Comment:
Is the case balancer and all subchannel connections lost before timeout expires covered in tests?

Reply:
Yes. It's covered in `grpclbFallback_initialTimeout_serverListReceivedBeforeTimerExpires()`. On `GrpclbLoadBalancerTest.java` line 1308 we close the LB stream, and when `timerExpires == false`, everything in this test happens before the timeout expires.

Reply:
I'm asking about the case no serverListReceived (thus the timer is not cancelled), and balancer and all subchannel connections lost before timeout expires, (pending task number should still be 1), and then when timer expires, fallback is used.

Reply:
Maybe the test name should be changed to grpclbFallback_initialTimeout_serverList**Not**ReceivedBeforeTimerExpires
because the comment says "not received"?

```java
@Test
  public void grpclbFallback_initialTimeout_serverListReceivedBeforeTimerExpires() {
    subtestGrpclbFallbackInitialTimeout(false);
  }

  @Test
  public void grpclbFallback_initialTimeout_timerExpires() {
    subtestGrpclbFallbackInitialTimeout(true);
  }

  // Fallback within the period of the initial timeout, where the server list is not received from
  // the balancer.
  private void subtestGrpclbFallbackInitialTimeout(boolean timerExpires) {
    long loadReportIntervalMillis = 1983;
```

Reply:
No problem! I added a LB stream breakage before timer expires.

Reply:
The comment was not accurate. I kept the test name and rephrased the comment.